src/test-utils.tsx

```diff
@@ -21,19 +21,21 @@ import { ConfigProvider } from 'antd'
 import { ReactElement } from 'react'
 
 import { ThemeContext, defaultTheme } from './components/ThemeProvider'
+import { ANT_PREFIX } from './components/ThemeProvider/Ant'
 
 /**
  * Wraps a React element with a series of providers for testing purposes.
  *
  * @remarks It adds the default theme to components
+ * @remarks It adds the default Ant class prefix
  * @remarks It removes the css-do-not-override classes from Ant Design components
  *
  * @param {ReactElement} children - The React element to be wrapped.
  * @returns {ReactElement} The wrapped React element.
  */
 const TestWrapper = ({ children }: {children: ReactElement}): ReactElement => (
```

Comment:
Mettiamo un riferimento su storybook di questa utils sul componente del ThemeProvider? Così l'utente legge scopre il setup e sa da subito che esiste il suo mock.

Nel caso non ci fosse uno storybook, Creiamo in cima un Getting started per il design system? 
Sempre come se fosse una stories che descrive come usare praticamente il DS.
Che ne pensi?

Reply:
Yes, I would add a Getting Started landing page, where every basic concept behind DS is explained, starting from the ThemeProvider, themes configuration, and tests configuration. 

 I created a specific issue to address this problem: #321